bokehjs/src/lib/models/widgets/div.ts

```diff
@@ -1,14 +1,36 @@
 import {Markup, MarkupView} from "./markup"
 import * as p from "core/properties"
+import {BundleProvider, MathJaxProvider} from "models/text/providers"
+import {find_math_parts} from "models/text/utils"
+import {TeX} from "models"
```

Comment:
You've just made `models/widgets/div` depend on **all** other model modules. Use a specific import.